src/index.js

```diff
@@ -1,241 +1,267 @@
-//#############################################################################
-//#
-//# "SQIP" (pronounced \skwɪb\ like the non-magical folk of magical descent)
-//# is a SVG-based LQIP technique - https://github.com/technopagan/sqip
-//#
-//# Installation:
-//# npm install -g sqip
-//#
-//# Requirements:
-//# * Node.js >= v.6 (https://nodejs.org/en/)
-//# * Golang (https://golang.org/doc/install)
-//# * Primitive (https://github.com/fogleman/primitive)
-//#
-//#############################################################################
-
-
-//#############################################################################
-//# CONFIG
-//#############################################################################
+// #############################################################################
+// #
+// # "SQIP" (pronounced \skwɪb\ like the non-magical folk of magical descent)
+// # is a SVG-based LQIP technique - https://github.com/technopagan/sqip
+// #
+// # Installation:
+// # npm install -g sqip
+// #
+// # Requirements:
+// # * Node.js >= v.6 (https://nodejs.org/en/)
+// # * Golang (https://golang.org/doc/install)
+// # * Primitive (https://github.com/fogleman/primitive)
+// #
+// #############################################################################
+
+// #############################################################################
+// # CONFIG
+// #############################################################################
 
 // Require the necessary modules to make sqip work
-const sizeOf = require('image-size');
-const argv = require('argv');
-const os = require('os');
-const child_process = require('child_process');
-const fs = require('fs');
-const path = require('path');
-const svgo = require('svgo');
+const sizeOf = require('image-size')
+const argv = require('argv')
+const os = require('os')
+const childProcess = require('child-process')
```

Comment:
I think it is still wrong ? :smile:

Reply:
Also eslint will fail on `camelcase` I assume, for that we can disable check for that line.

Reply:
I am not sure what u mean here. Camelcase is enforced for variable names, not for module names/string content 🤔

Reply:
Looks like it is not failing on ESLint as you said sorry :) 

But `child_process` module is with underscore instead of dash.

Reply:
Omg. Sure! Note to myself: Test the code before pushing.

Usually I do that but since we plan do do more for the next version I actually did not test this commit. Sorry